src/main/java/org/sefglobal/scholarx/service/EmailService.java

```diff
@@ -0,0 +1,23 @@
+package org.sefglobal.scholarx.service;
+import org.sefglobal.scholarx.model.Email;
+import org.sefglobal.scholarx.util.EmailUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class EmailService {
+
+    @Autowired
+    private EmailUtil emailUtil;
+
+    public Email sendEmail(String email, String subject, String message){
+        Email emailObj = new Email();
```

Comment:
can you rename the `emailObj` to `email`?
because the object type is already there

Reply:
@anjula-sack, I updated the pull request, can you check?